consensus/module.go

```diff
@@ -109,6 +112,38 @@ func (m *consensusModule) SetUtilityContext(utilityContext modules.UtilityContex
 	m.utilityContext = utilityContext
 }
 
+// Implementations of the ConsensusStateSync interface
+func (m *consensusModule) GetLeaderId() uint64 {
```

Comment:
Where is `GetLeaderId` being used?

Reply:
currently nowhere so I removed, thanks for catching. I will add if it turns out necessary for additional functionality.